applications/rubintv/values-usdfdev.yaml

```diff
@@ -0,0 +1,89 @@
+siteTag: "usdf-dev"
+
+frontend:
+  debug: true
+  env:
+  - name: http_proxy
+    value: "http://sdfproxy.sdf.slac.stanford.edu:3128"
+  - name: https_proxy
+    value: "http://sdfproxy.sdf.slac.stanford.edu:3128"
+  - name: no_proxy
+    value: ".sdf.slac.stanford.edu,.slac.stanford.edu,localhost,127.0.0.1"
```

Comment:
I feel like SLAC found some way to not have to embed these settings in every application, but I don't remember what it was.

Reply:
If they did I have no memory of it.  I've asked on Slack.

Reply:
Apparently they are not required but do speed up things?  I'll change the frontend to remove them, anyway.

Reply:
And workers too.  We can put 'em back if we really need them.